app/assets/javascripts/discourse/app/mixins/composer-upload-uppy.js

```diff
@@ -447,42 +448,8 @@ export default Mixin.create(ExtendableUploader, UppyS3Multipart, {
         );
       });
 
-    this._onPreProcessProgress((file) => {
```

Comment:
I think we could still leave this off, and just change this code:

https://github.com/discourse/discourse/blob/bb342bafe9a9d41f912c49e2eeda3edb66c60b11/app/assets/javascripts/discourse/app/mixins/composer-upload-uppy.js#L304-L309

To use the `processingPlaceholder` from the beginning...then when `_onPreProcessComplete` is fired replace it with the uploading placeholder